src/extension.ts

```diff
@@ -293,6 +293,11 @@ async function activate(context: vscode.ExtensionContext) {
         );
     },
     togglePrettyPrint: eval.togglePrettyPrint,
+    activateCalva: () => {
+      return new Promise((resolve, _reject) => {
+        resolve(true);
+      });
```

Comment:
What's the purpose of this? @PEZ

Reply:
I should have left some comments there! This is so that the command 'activateCalva' can be used to, well, activate Calva. And you can await the command and know that now Calva can be required and expected to work. We use it before running any tests.

Reply:
It looks like it just returns a Promise that resolves to true, rather than doing any activating of Calva. What am I missing?

Reply:
You are missing (because I failed to add a comment about it) that this function is registered as an activation command in the manifest. So calling it activates Calva, and the promise makes it possible to await. I think it is in runTests.js where we call it. (I'm afk)